tests/behat/uploadFileToMoodle.feature

```diff
@@ -0,0 +1,27 @@
+@ocis @javascript
+Feature: upload the resource in oCIS to moodle
+  As a user
+  I want to list the resources
+  So that I can upload them to moodle
+  
+
+  Background:
+    Given I log in as "admin"
+    And I follow "Profile" in the user menu
+    And I click on "Blog entries" "link"
+    And I click on "Add a new entry" "link"
+    And I click on "Add.." "button"
+    And I click on "oCIS" "link"
+    And I click on "Log in to your account" "button"
+    And I switch to a second window
+    And I set the field with xpath "//*[@id='oc-login-username']" to "admin"
+    And I set the field with xpath "//*[@id='oc-login-password']" to "admin"
+    And I click on "Log in" "button"
+    And I click on Allow Button
```

Comment:
create a single new step
`Given I log in to ocis as "admin" to ocis`

Reply:
Maybe just 
`Given I log in to ocis as "admin"`
or
`Given I log in as "admin" to ocis`